docs/source/notebooks/tpt.ipynb

```diff
@@ -0,0 +1,779 @@
+    "    \n",
+    "To demonstrate the TPT API ([API docs here](../api/generated/sktime.markov.ReactiveFlux.rst#sktime.markov.ReactiveFlux)), in the following the example of a drunkard's walk is presented. The example is motivated by <cite data-cite=\"nbtpt-doyle1984random\">(Doyle, 1984)</cite> and <cite data-cite=\"nbtpt-valleriani2015circular\">(Valleriani, 2015)</cite>, where a drunkard is placed on a network of states and two special states, home and the bar. When the drunkard reaches either of these special states the trajectory stays there with high probability.\n",
+    "One can then ask which paths can be taken and also with which probability the drunkard is going to reach either of the states given a certain current position."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To this end, import sktime and numpy for general numerical operations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import sktime"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can create a [DrunkardsWalk](../api/generated/sktime.data.drunkards_walk.rst#sktime.data.drunkards_walk) simulator by specifying bar and home locations. As the drunkard lives on a 2-dimensional surface, the locations are given in terms of integer coordinates. Internally, this is related back to $\\mathrm{width}\\times\\mathrm{height}$ states. On a 'normal' state (i.e., not home/bar or barrier), the drunkard can stagger a step within a $3\\times 3$ grid with uniform probability without staying on the same spot."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sim = sktime.data.drunkards_walk(grid_size=(10, 10),\n",
+    "                                 bar_location=[(0, 0), (0, 1), (1, 0), (1, 1)], \n",
+    "                                 home_location=[(8, 8), (8, 9), (9, 8), (9, 9)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To make the scenario a bit more interesting, we can add hard and soft barriers by specifying start and end points of the barrier. If no weight is given, the barrier is `hard`, i.e., cannot be crossed by a trajectory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sim.add_barrier((5, 1), (5, 5))\n",
+    "sim.add_barrier((0, 9), (5, 8))\n",
+    "sim.add_barrier((9, 2), (7, 6))\n",
+    "sim.add_barrier((2, 6), (5, 6))\n",
+    "\n",
+    "sim.add_barrier((7, 9), (7, 7), weight=5.)\n",
```

Comment:
I would've thought weights go between 0 and 1. Can you give an intuition for weight values

Reply:
done

Reply:
thank you for adding the sentence!